packages/skeleton/src/skeleton.tsx

```diff
@@ -65,8 +65,19 @@ const fade = keyframes({
   to: { opacity: 1 },
 })
 
+const useIsFirstRender = () => {
+  const isFirstRender = React.useRef(true)
+
+  React.useEffect(() => {
+    isFirstRender.current = false
+  }, [])
+
+  return isFirstRender.current
+}
```

Comment:
Should this be moved to a "utils" file? or ok here for now until we need to reuse it?

Reply:
We'll leave it here for now. When we need to reuse it, I'll move it.

Thanks buddy